src/getTime.js

```diff
@@ -18,7 +18,14 @@
  * @returns {string} - valid time
  */
 function getTime(str) {
-  // write code here
+  const pattern = /\b([0-1][0-9]|[2][0-3]):\b[0-5][0-9]\b/g;
+  const time = str.match(pattern);
```

Comment:
мы получаем список совпадений, почему time?
```suggestion
  const matches = str.match(pattern);
```